test/fixtures/nullProps/serverless.yaml

```diff
@@ -0,0 +1,16 @@
+service: stripNullPropsFromTemplateResourcesTest
+
+provider:
+  name: aws
+
+resources:
```

Comment:
I think that in this particular case, we don't need to have a separate `fixture` for purposes of only one test - let's remove it and instead let's use e.g. `aws` fixture extended with `configExt` in `runServerless`